metron-deployment/Kerberos-manual-setup.md

```diff
@@ -382,18 +380,18 @@ Push Data
 1. Push some sample data to one of the parser topics. E.g for Bro we took raw data from [metron/metron-platform/metron-integration-test/src/main/sample/data/bro/raw/BroExampleOutput](../metron-platform/metron-integration-test/src/main/sample/data/bro/raw/BroExampleOutput)
 
     ```
-  	cat sample-bro.txt | ${KAFKA_HOME}/kafka-broker/bin/kafka-console-producer.sh \
-  	        --broker-list ${BROKERLIST}
-          	--security-protocol SASL_PLAINTEXT \
+    cat sample-bro.txt | ${KAFKA_HOME}/kafka-broker/bin/kafka-console-producer.sh \
+            --broker-list ${BROKERLIST}
```

Comment:
Unrelated to your change, but you throw a '\' at the end of this line while you're in here?